### PR TITLE
chore: foundations typography heading h1   h6

### PR DESCRIPTION
**Pull request by bashirkarimi**

### CODE REVIEW

**Comment by oliverschuerch** [comment]
Depends on #3543

File: packages/styles/src/elements/heading.scss [hunk 1]
```diff
@@ -0,0 +1,62 @@
+@use '../functions/tokens' as tokens;
+@use '../tokens/elements' as elements;
```
Comment (gfellerph) [inline packages/styles/src/elements/heading.scss:2]: With `@use`, the file name automatically becomes the namespace. Overriding the namespace name with `as X` is only necessary when the filename is not optimal or two identically named files are being imported.

```suggestion
@use '../functions/tokens';
@use '../tokens/elements';
```

File: .changeset/eleven-keys-work.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+'@swisspost/design-system-documentation': patch
+'@swisspost/design-system-styles': patch
+---
+
+Updated the style of Headings with design token
```
Comment (gfellerph) [inline .changeset/eleven-keys-work.md:6]:
```suggestion
Updated the style of Headings (h1-h6) using design tokens.
```

File: .changeset/eleven-keys-work.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+---
+'@swisspost/design-system-documentation': patch
+'@swisspost/design-system-styles': patch
+---
+
+Updated the style of Headings (h1-h6) using design tokens.
```
Comment (alizedebray) [inline .changeset/eleven-keys-work.md:6]: I suggest not to mention the tokens in the change log, as they are part of our internal technology that users don't necessarily need to be aware of:
```suggestion
Updated the style of headings (h1-h6).
```

**Review by alizedebray — APPROVED** [review]

**Review by gfellerph — APPROVED** [review]

**Pull request merged**